homeassistant/components/airzone_cloud/climate.py

```diff
@@ -262,6 +285,9 @@ async def async_set_hvac_mode(self, hvac_mode: HVACMode) -> None:
 class AirzoneAidooClimate(AirzoneAidooEntity, AirzoneDeviceClimate):
     """Define an Airzone Cloud Aidoo climate."""
 
+    _speeds: dict[int, str] = {}
+    _speeds_reverse: dict[str, int] = {}
```

Comment:
Please set these in the constructor, having mutable dicts outside of the constructor can have unwanted side effects

Reply:
In fact we don't need to init them here, since `_set_fan_speeds` is already doing that, so I will remove the `= {}` from here.
Sorry, I just copied this from the airzone integration: https://github.com/home-assistant/core/blob/c83388fd2da3fab062dec3fc3c3ebc8013ca478b/homeassistant/components/airzone/climate.py#L118-L119

Reply:
> In fact we don't need to init them here, since `_set_fan_speeds` is already doing that, so I will remove the `= {}` from here. Sorry, I just copied this from the airzone integration:
> 
> https://github.com/home-assistant/core/blob/c83388fd2da3fab062dec3fc3c3ebc8013ca478b/homeassistant/components/airzone/climate.py#L118-L119

Done in cdbe317185844534075504d610aec0a92beb640e